neo/Network/RPC/RpcServer.cs

```diff
@@ -90,12 +90,17 @@ private JObject GetInvokeResult(byte[] script)
             return json;
         }
 
-        private static JObject GetRelayResult(RelayResultReason reason)
+        private static JObject GetRelayResult(RelayResultReason reason, UInt256 hash)
         {
             switch (reason)
             {
                 case RelayResultReason.Succeed:
-                    return true;
+                    {
+                        var ret = new JObject();
+                        ret["result"] = true;
+                        ret["hash"] = hash.ToString();
+                        return ret;
```

Comment:
I can't quickly tell but will this result in 
```
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": true,
  "hash": "some tx id hash"
}
```
or in
```
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
     "result": true,
     "hash": "some tx id hash"
   }
}
```

?

Reply:
The second one, but i will change result for 'relayed'